.github/ISSUE_TEMPLATE/observability_issue.yml

```diff
@@ -0,0 +1,88 @@
+name: Observability issue
+description: File an issue about telemetry, alert rules or dashboards.
+labels: ["Type: Bug", "Status: Triage"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thanks for taking the time to report an observability issue!
+        Before submitting your issue, please inspect recent issues and pull requests,
+        to make sure it's not already solved.
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: >
+        What signal/alert you expected to see and under what conditions
+        (metric, log pattern, trace span, threshold, duration),
+        why/how an alert should have fired sooner, etc.
+    validations:
+      required: true
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual Behavior
+      description: >
+        What happened? What symptoms did you observe? Root cause discovered?
+        Did the problem happen in particular time windows? What was the impact?
+        Were there false positives? What telemetry is missing?
+    validations:
+      required: true
+  - type: textarea
+    id: telemetry-available
+    attributes:
+      label: Relevant telemetry / alerts available
+      description: >
+        Please copy and paste any relevant log, metrics, screenshots of timeseries.
+    validations:
+      required: true
+  - type: textarea
+    id: telemetry-missing
+    attributes:
+      label: Missing or insufficient telemetry / alerts
```

Comment:
This is kinda already included in the above. Previously you ask 

> What signal/alert you expected to see

> Were there false positives? What telemetry is missing?

And then here:
> Specific metrics / labels missing

Redundant required fields are quite annoying when trying to report bugs.